aiojenkins/builds.py

```diff
@@ -177,8 +178,10 @@ async def start(self,
                 }),
                 **parameters,
             }
+
+            path += "/buildWithParameters"
         else:
-            path = '/job/{}/build'.format(name)
```

Comment:
the issue is that if `name` contains folder, path should be `{folder_name}/job/{job_name}`. otherwise, the URL path is not correct.

Reply:
okay, it's clear now